### ISSUE TITLE
'Edit Problem' modal does not receive keyboard focus

### ISSUE THREAD

**Issue by Sanjog1605** [benetech/MathShare#1525]
## Issue Description
When we edit any pre-existing problem within a problem set, the modal does not receive keyboard focus.

**Steps to reproduce:**

1. Go to [Solve for X](https://mathshare-qa.diagramcenter.org/#/app/problemSet/edit/BIH2267I7IIFU) webpage on Mathshare.
2. Click on 'more options' button under "Solve for X".
3. Click on "Edit problem".

**Expected Result:**
The keyboard focus should land in the modal box.

**Actual Result:**
The keyboard focus remains in the background, and when the 'escape' key is pressed, the modal closes but the 'more options' menu remains in the expanded state.

**Screenshots:**
![The modal dialog box has opened but the keyboard focus is in the background](https://user-images.githubusercontent.com/68078108/98675505-5e4fb480-2380-11eb-9e4f-708099b7e7f7.png)

**Testing Environment:**
 - OS: Windows 10 Pro v20h2
 - Browser: Mozilla Firefox 82.0.2 (x64)
 - Screen Reader: NVDA 2020.3

**Comments**
@alexrcabral  sir, in addition to this issue, I found that some of the more complex mathematical symbols are missing, such as, integration sign, epsilon etc.
